go/vt/sqlparser/ast_format.go

```diff
@@ -326,6 +326,11 @@ func (node *ShowThrottledApps) Format(buf *TrackedBuffer) {
 	buf.astPrintf(node, "show vitess_throttled_apps")
 }
 
+// Format formats the node.
+func (node *ShowThrottlerStatus) Format(buf *TrackedBuffer) {
+	buf.astPrintf(node, "show vitess_throttler status")
```

Comment:
Now sue how you mean? this shows the status of the throttler on all shards

Reply:
I would expect stats for STATUS values in MySQL and VARIABLES for configuration. Isn't this showing the config rather than status? That's the naming you used elsewhere and IMO the values returned from this statement are the current throttler configuration on each tablet rather than operational status (is it currently engaged, what's currently throttled, how many times throttler has kicked in, etc).

Reply:
I still think the term `status` is correct. You configure the throttler to be something -- by writing to `topo`; but then you check on the tablets to see what they actually think; you have configured the throttler to be `--enabled`. But are the tablets's throttlers actually enabled?

I think this is similar to setting/monitoring semi-sync replication: you `set rpl_semi_sync_source_enabled`, but then you check for `rpl_semi_sync_source_status`.

I don't have strong opinions and don't really care how the command is named. Please let me know your preference and I'll adapt.

Reply:
Ok, we can leave it as-is. I wouldn't say that there's really an objective right or wrong here (e.g. `SHOW REPLICA STATUS` is a mix of configs and stats). It's your work so we can stick with your preference.